hijackers/utls.go

```diff
@@ -3,26 +3,29 @@ package hijackers
 import (
 	"crypto/tls"
 	"fmt"
-	"github.com/fedosgad/mirror_proxy/utils"
-	utls "github.com/refraction-networking/utls"
```

Comment:
Import ordering and other cosmetic changes are out of scope of this PR - please rollback to avoid PR overload.